README.md

```diff
@@ -26,57 +28,57 @@ If this repo contains any reference architectures, uncomment the heading below a
 See "Reference architecture" in Authoring Guidelines in the public documentation at
 https://terraform-ibm-modules.github.io/documentation/#/implementation-guidelines?id=reference-architecture
 -->
-<!-- ## Reference architectures -->
 
+## Reference architectures
+
+[![Maximo Application Suite Deployable Architecture](https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-mas/main/reference-architecture/mas_deployable_architecure.svg)]
 
 <!-- This heading should always match the name of the root level module (aka the repo name) -->
 ## terraform-ibm-mas
 
 ### Usage
 
-<!--
-Add an example of the use of the module in the following code block.
-
-Use real values instead of "var.<var_name>" or other placeholder values
-unless real values don't help users know what to change.
--->
+You can use the modular design of this module to provision Maximo Application Suite Core offering or Maximo Application Suite Core + Manage offering.
 
 ```hcl
 
+module "maximo" {
+
+  source                       = "terraform-ibm-modules/mas/ibm"
+  version                      = "X.X.X" # Replace "X.X.X" with a release version to lock into a specific release. For example, 1.6.1
+  cluster_id                   = var.cluster_id
+  deployment_flavor            = var.deployment_flavor # For example, core
+  mas_entitlement_key          = var.mas_entitlement_key
```

Comment:
this was renamed to `entitlement_key`

Reply:
Sorry..this README.MD file changes are somehow missed..fixed it..thanks